extensions/objects/templates/list-of-sets-of-html-strings-editor.directive.ts

```diff
@@ -93,10 +94,6 @@ angular.module('oppia').directive('listOfSetsOfHtmlStringsEditor', [
           return errorMessage;
         };
         ctrl.$onInit = function() {
-          if (!ctrl.selectedRank) {
-            ctrl.selectedRank = '';
```

Comment:
Replace this by setting all `choice.selectedRank = ''`, as right now, aren't they undefined?

Reply:
Ah, yes! I missed this. I've added it now.